index.html

```diff
@@ -0,0 +1,328 @@
+            <div class="grid-item item2">
+                <div class="content">
+                    <h1 class="p-text">Professional Art <br>
+                        Printing Data
+                    </h1>
+                    <h2 class="s-text">
+                        A daily selection of privately
+                        personalized reads; no accounts or
+                        sign-ups required. Has been the 
+                        industry's standard.
+                    </h2>
+                    <div class="tags">
+                        <ul class="tag">
+                            <li> <a href="#">HTML</a></li>
+                            <li> <a href="#">Bootstrap</a></li>
+                            <li> <a href="#">Ruby</a></li>
+                        </ul>
+                    </div>
+
+                    <div class="btn2-holder">
+                        <a href="#" class="btn2">See Projects</a>
+                    </div>
+                </div>
+            </div>
+            <div class="grid-item item2">
+                <div class="content">
+                    <h1 class="p-text">Professional Art <br>
+                        Printing Data
+                    </h1>
+                    <h2 class="s-text">
+                        A daily selection of privately
+                        personalized reads; no accounts or
+                        sign-ups required. Has been the 
+                        industry's standard.
+                    </h2>
+                    <div class="tags">
+                        <ul class="tag">
+                            <li> <a href="#">HTML</a></li>
+                            <li> <a href="#">Bootstrap</a></li>
+                            <li> <a href="#">Ruby</a></li>
+                        </ul>
+                    </div>
+
+                    <div class="btn2-holder">
+                        <a href="#" class="btn2">See Projects</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <section class="main-container">
+            <article class="about-container">
+                <article class="about">
+                        <h1 class="about-p-text" >About Me</h1>
+                        <p class="about-s-text">Hello I’m a software developer! I can 
+                            help you build a product , feature or 
+                            website Look through some of my work 
+                            and experience! If you like what you 
+                            see and have a project you need coded, 
+                            don’t hesitate to contact me.
+                        </p>
+                        <div class="btn-holder">
+                            <button class="btn">Get My Resume</button>
+                        </div>
+                        
+                        <img class="about-me-banner" src="/assets/images/About Me Background.svg" alt="Aboutmebanner">
```

Comment:
> - [ ] Please kindly use a background image for decorator images instead of an `img` tag. This is for aesthetic reasons, You can [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.👍